common/src/main/java/apoc/trigger/TriggerHandler.java

```diff
@@ -314,4 +316,4 @@ private void setLastUpdate(Transaction tx) {
         node.setProperty(SystemPropertyKeys.lastUpdated.name(), System.currentTimeMillis());
     }
 
-}
+}
```

Comment:
I was wondering if you knew the reason why TriggerHandler is in `common` whereas all the other functionality (including TriggerHandlerNewProcedures) is in `core`?

Reply:
I honestly don't know, most likely an error when core and extended were split in 2 repos, probably because there is a `TriggerExtended`, which however does not use this class.

I moved this class (and also `TriggerMetadata`, placed in the same package),
and everything seems to be working fine.

Reply:
You may need to check that APOC Extended still builds successfully after moving it. But I don't see why the functions in Extended would depend on this TriggerHandler file.